LICENSE

```diff
@@ -1,21 +1,30 @@
-MIT License
+Copyright Author name here (c) 2018
 
-Copyright (c) 2018 Holmusk
```

Comment:
I see that LICENSE is changed. Could you please return back to previous license?